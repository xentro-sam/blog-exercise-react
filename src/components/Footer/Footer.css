.footer {
    background-color: #000000;
    color: white;
}

.footer-content {
    display: flex;
}
