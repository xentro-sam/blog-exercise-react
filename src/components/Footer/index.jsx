import * as React from 'react';
import './Footer.css';

export default function Footer() {
  return (
    <div className="footer page-padding">
      <div className="footer-content">
        <h4>&copy;artifact.com 2019</h4>
      </div>
    </div>
  );
}
