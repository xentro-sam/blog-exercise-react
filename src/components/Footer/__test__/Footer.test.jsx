import React from 'react';
import {render} from '@testing-library/react';
import Footer from '..';

describe('Footer', () => {
  it('should render correctly', () => {
    const {asFragment} = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
  it('should render the title', () => {
    const {queryByText} = render(<Footer />);
    expect(queryByText('Â©artifact.com 2019')).toBeTruthy();
  });
});
