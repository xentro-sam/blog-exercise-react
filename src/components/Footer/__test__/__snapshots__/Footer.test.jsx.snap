// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<DocumentFragment>
  <div
    class="footer page-padding"
  >
    <div
      class="footer-content"
    >
      <h4>
        Â©artifact.com 2019
      </h4>
    </div>
  </div>
</DocumentFragment>
`;
